src/types/buffer/mod.rs

```diff
@@ -22,43 +22,40 @@ pub trait TypedArray: private::Sealed {
     ///
     /// This may not be used if a mutable borrow is in scope. For the dynamically
     /// checked variant see [`TypedArray::try_borrow`].
-    fn as_slice<'a: 'b, 'b, C>(&'b self, cx: &'b C) -> &'b [Self::Item]
+    fn as_slice<'cx, 'a, C>(&self, cx: &'a C) -> &'a [Self::Item]
```

Comment:
`'a` is renamed to `'cx` across the board for clarity. `'b` becomes `'a` since that's the only lifetime we need.